src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1815,6 +1815,14 @@ private int getStringLiteral(CORINFO_MODULE_STRUCT_* module, uint metaTOK, char*
             return str.Length;
         }
 
+#pragma warning disable CA1822 // Mark members as static
+        private int objectToString(void* handle, char* buffer, int size)
+#pragma warning restore CA1822 // Mark members as static
+        {
+            // TODO: implement
```

Comment:
`var obj = HandleToObject((IntPtr)handle);` will give you the compiler object, and the rest should be simple.